### ISSUE TITLE
Crashing!

### ISSUE THREAD

**Issue by felki12** [NMan1/Rainbow-Six-Cheat#10]
I could build the driver and the hack itself flawlessly, after that I dumped the game and I updated the offsets using IDA, the glow func and no clip, and fov works perfectly but when I try to enable No recoil/Spread or speed or so nothing happens. I figured out that when I load a config everything gets enabled and then I see that my Crosshair gets together meaning that the no spread is enabled but this happen in a second or so and the game crashes! Help-me I am trying to learn how this cheat works but I do not know why its crashing or so! I also figured out that when I run a shell code from the unlock all blocking the game to crash everything works and the game doesnt crash. but I couldn't make the Shell code work inside the driver So its only when i inject the unlock all dll together.

**Comment by NMan1** [comment]
I’d bet the crash is caused from some feature (could be no recoil or unlock all) writing to the .text section without patching TerminateProcess correctly, so check that. 

Also the shell code for patching unlock all definitely has changed. 

The game has probably updated upwards of 20 times since this cheat was functional (hence the wrong shell code) so the problem could really be anything. It’s up to you to debug it.

**Comment by felki12** [comment]
Yeah, i thought that. cause when I use the shellcode (only in lan cause its usermode WPM and i'm also testing the cheat offline before going online) It works. But one thing that i didnt figured out is why the options in the menu dont work for it self, like I can enable chams but if i click no recoil nothing happens. But if I click load rage/default/legit settings, everything gets enabled my cam turn arround and the spread works cause my crosshair gets together but it happens in a second or so cause after that the game crashes. Sorry for the english, not my language. only a newbie trying to learn xD thanks for the reply!

**Comment by JayneFrack** [comment]
Same happens with me did you find a fix?

**Comment by Aspect-So1ar** [comment]
Anyone have it fixed?

**Comment by O64198494894196** [comment]
@felki12 u make upload code pls? sorry my english
